Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -255,6 +266,13 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         tabManager.removeTab(tab)
         tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: UITableViewRowAnimation.Automatic)
     }
+    
+    func closeCurrTab(sender: UIButton) {
```

Comment:
Nit: Please rename this to `SELdidClickClose`. We prefix selectors with `SEL` so it's clear they're selectors.